implementations/ruby-json_schemer/Gemfile

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-gem 'json_schemer', '2.3.0'
+gem 'json_schemer', ENV.fetch('JSON_SCHEMER_VERSION', '2.3.0')
```

Comment:
I suspect it very well may ruin dependabot, but honestly I don't know how to check.

Reply:
> I suspect it very well may ruin dependabot, but honestly I don't know how to check.

Hmm, idk if I am getting too ahead of myself here but then should we ask this query to the repo maintainer then ? 😅

Reply:
> upto the version v0.2.17 of "json_schemer". Going back beyond this version fails the Dockerfile build process because of incompatible Ruby versions which is again something that would have to be taken care of

also this issue arises when trying to go back `v0.2.17`